doc/Type/Any.pod6

```diff
@@ -339,7 +339,7 @@ Interprets the invocant as a list, and returns the last index of that list.
 
 Defined as:
 
-    multi method pairup(Any:U:)
+    multi method pairup(Any:U: --> Seq)
```

Comment:
I really want to keep the description consistent with the source, and this is not the case here:
https://github.com/rakudo/rakudo/blob/2ff4a88b5b0018c490777aa7eceebed4b000a62a/src/core/Any-iterable-methods.pm6#L1891-L1892
When that is not the case, it's impossible to know if it's code drift or simply an error. So keeping it consistent is the only way to keep it maintenable. You can always add what it returns in the description. So I would really prefer this change to be reverted.

Reply:
Understood.